.github/workflows/shadow-repo.yml

```diff
@@ -0,0 +1,72 @@
+name: Sync with shadow repo
```

Comment:
This can be removed if you like, but for development on static sites we've found this to be a very useful workflow. Basically we create a separate repo (e.g. `sass-site-built`), and add the required vars/secrets to this repo, and then every PR gets a link showing the full diff of actual compiled files.

See: https://github.com/oddbird/sass-site/pull/58#issuecomment-1570327910